src/server/pfs/server/trigger.go

```diff
@@ -96,16 +96,18 @@ func (d *driver) isTriggered(txnCtx *txncontext.TransactionContext, t *pfs.Trigg
 		}
 	}
 	if t.Size_ != "" {
-		size, err := units.FromHumanSize(t.Size_)
+		_, err := units.FromHumanSize(t.Size_)
 		if err != nil {
 			// Shouldn't be possible to error here since we validate on ingress
 			return false, errors.EnsureStack(err)
 		}
-		var oldSize uint64
-		if oldHead != nil {
-			oldSize = oldHead.SizeBytes
-		}
-		merge(int64(newHead.SizeBytes-oldSize) >= size)
+		// TODO(2.0 required): the size of a commit isn't known when finishing the
```

Comment:
This also turned into a segfault because we weren't getting the `CommitInfo.Details` here (the commit has just finished and we don't want to wait for compaction).

Reply:
How would triggers work with deletions? It looks like it only triggers if the commit size gets larger (in the size case), rather than the size of new data since the last commit. It might make sense to have a notion of total referenced size in a commit chain, which might make more sense for trigger semantics and would give us a low cost way to provide users with feedback that their data is going into a commit without computing the commit size.

Reply:
I think we should look into storing an accumulator on the branch trigger that we update after each change so we don't have to guess at the differences at this point in the code.  For example, it walks backwards from the head of the source branch to find a common ancestor, but that stops working if you don't make linear commits in the source branch.  Maybe we could do a similar thing with the diff size?